### PR TITLE
Spanish fully-translated

### PR DESCRIPTION
**Pull request by saifulislamdev**

### CODE REVIEW

File: src/markdown-pages/landing/landing-es.md [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
 templateKey: landing-page
 slug: home
-pageTitle: es Project | Home
+pageTitle: Proyecto | Inicio
 hero:
-  heading: es Customizable Landing Page Template 
-  text: es Set up a landing page for your agency with this template. It includes all of the resources that you need to have an easy, compliant, secure, appealing, and sustainable landing page.
+  heading: Plantilla Editable para Página de Inicio 
+  text: Crea una página de inicio para tu institución de gobierno con esta plantilla. Incluye todos los recursos que puedes necesitar para tener una página de inicio simple, segura, atractiva y sostenible. 
   buttonLink: https://github.com/nyc-cto/LPaaS2/
```
Comment (aannyyaa) [inline src/markdown-pages/landing/landing-es.md]: `buttonLink` for github and contact can be moved to links constants

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -9,28 +9,28 @@
     "dropdowns": [
       {
         "buttonLabel": "Nav Label 1",
```
Comment (aannyyaa) [inline src/locales/en/translation.json]: We didn't get translations for nav and parent link labels?

File: src/markdown-pages/landing/landing-es.md (hunk 1 above)
Comment (saifulislamdev) [reply src/markdown-pages/landing/landing-es.md]: Hero and section button links moved to `src/constants/links.js`.

File: src/locales/en/translation.json (hunk 2 above)
Comment (saifulislamdev) [reply src/locales/en/translation.json]: The translator kept it the same while leaving a comment regarding the parent link labels: "This is a coding term I think developers call this thing the same in Spanish."

**Review by aannyyaa — APPROVED** [review]

**Pull request merged**